### PR TITLE
[source-sentry] [source-sendgrid] [source-greenhouse] add .yaml to list of file imports in setup.py

### PR DESCRIPTION
**Pull request by brianjlai**
## What
Addresses https://github.com/airbytehq/oncall/issues/462

We're getting file not found errors on the container on Cloud when we try to run syncs with declarative connectors. This is likely because we don't specify that .yaml files in the working directory are usable by the program. I'm going to test this with one source first, and if it works, then I'll roll it out for the other two.

## How
Adds `*.yaml` to `package_data` so that it can be used at runtime by the declarative source being executed

### CODE REVIEW

**Comment by sherifnada** [comment]
@brianjlai how does this pass SAT but then fail on cloud? 🤔

**Comment by brianjlai** [comment]
That I'm not sure about. I haven't been able to replicate this locally when I package up a connecter on my own machine either. But it is also the only thing I can think of that could cause the file to not be read, since we copy the whole directory over.

I wouldn't expect this to work in any capacity. Figuring out why can be part of a post-fix retro given that this affects all 3 connectors

**Comment by brianjlai** [comment]
I think this is a pretty safe change to publish and test, given that there are no code changes, and a minimal config adjustment that we have configured for other connectors in prod

**Review by sherifnada — COMMENTED** [review]
publishing this is fine w me but feels like a total shot in the dark. If the connector works locally then that means the yaml _was_ packaged, no? is the problem an issue of relative paths? Does this connector work on a local instance of airbyte for you?

File: airbyte-integrations/connectors/source-greenhouse/integration_tests/expected_records.txt [hunk 1]
```diff
@@ -69,9 +69,9 @@
...
-{"stream":"demographics_questions","data":{"translations":[{"name":"Do you identify as transgender? (select one)","language":"en"}],"required":false,"name":"Do you identify as transgender? (select one)","id":4015598003,"demographic_question_set_id":4002702003,"answer_type":"multi_value_single_select","active":true},"emitted_at":1660156527389}
+{"stream":"demographics_questions","data":{"translations":[{"name":"Are you a veteran or active member of the United States Armed Forces?","language":"en"}],"required":false,"name":"Are you a veteran or active member of the United States Armed Forces?","id":4015594003,"demographic_question_set_id":4002702003,"answer_type":"multi_value_single_select","active":true},"emitted_at":1660156527389}
```
Comment (brianjlai) [inline airbyte-integrations/connectors/source-greenhouse/integration_tests/expected_records.txt:72]: I'm not sure what exactly happened here, but in the last week greenhouse might've changed their default question vernacular, but it no longer lines up with the expected records from last week. I doubled checked this against making the direct query via Postman and it does not include `(select one)` in the response so I'm confident this was a change on their end or in our test data

File: airbyte-integrations/connector-templates/source-configuration-based/setup.py.hbs [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk~=0.1.76",
+    "airbyte-cdk~=0.1.79",
```
Comment (brianjlai) [inline airbyte-integrations/connector-templates/source-configuration-based/setup.py.hbs:9]: technically this doesn't exist yet, but I will merge https://github.com/airbytehq/airbyte/pull/15814 first which will should fix retrieving schemas while running from a job

Comment (brianjlai) [reply airbyte-integrations/connector-templates/source-configuration-based/setup.py.hbs:9]: it now exists as of 8/22/22

**Comment by brianjlai** [comment]
This latest release of greenhouse includes the fix for both the config and schemas read issues. Version `0.2.9` I will deploy to cloud.

And rather than publish two new images under this same PR that already published, I'm gonna open a new PR to fix the remaining schema read issues by bumping the latest CDK. I validated those changes on a local version of Airbyte. This PR should be all set to merge in.

**Pull request merged**